src/logic/useWalkInfoBike.ts

```diff
@@ -44,7 +49,9 @@ async function getWalkInfoBike(
                         walkDistance: result[0].walkDistance,
                     }
                 })
-                .catch(() => null),
+                .catch((err) => {
+                    throw err
+                }),
```

Comment:
Lot denne bli her for å vise explisitt at erroren skal kastes videre. Kan sløyfes hvis det er ønskelig.